source/Handlebars/LayoutViewModel.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Collections;
+using System.Linq;
+using HandlebarsDotNet.Compiler;
+using HandlebarsDotNet.Iterators;
+using HandlebarsDotNet.MemberAccessors;
+using HandlebarsDotNet.ObjectDescriptors;
+using HandlebarsDotNet.PathStructure;
+
+namespace HandlebarsDotNet
+{
+    internal class LayoutViewModel
+    {
+        private static readonly ChainSegment _bodyChainSegment = ChainSegment.Create("body");
```

Comment:
Rename to `BodyChainSegment`: `static readonly` fields should be named starting with capital letter.
Add empty lime between `static` and `instance` members.